packages/file-server/env.example

```diff
@@ -0,0 +1,5 @@
+REMOTES="http://localhost:3001" # list of remote servers to proxy requests to (can be JSON array)
+PORT=3000 # port to run the server on
```

Comment:
I think the default port should be in line with the rest of our ports. We've used `9228` to `9231` so this one should perhaps be `9232`